packages/Rakefile

```diff
@@ -19,12 +19,9 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
   end
 
   def detect_target_apache_arrow_version
-    cmake_yml = File.join(__dir__,
-                          "..",
-                          ".github",
-                          "workflows",
-                          "cmake.yml")
-    File.read(cmake_yml)[/ref: apache-arrow-([\d.]+)/, 1]
+    latest_release_url = "https://api.github.com/repos/apache/arrow/releases/latest"
+    latest_tag_name = JSON.parse(URI.open(latest_release_url).read)["tag_name"]
+    latest_tag_name[/\Aapache-arrow-([\d.]+)\z/, 1]
```

Comment:
How about addition a example of latest_tag_name as comment? (e.g: # apache-arrow-19.0.0)

Reply:
fix: [227cd11](https://github.com/groonga/groonga/pull/2208/commits/227cd11678da62346c6c88e7d7fbe58117ed1b3d) Thanks. I've added an example here.